docs/guides/01-getting-started/prerequisites.md

```diff
@@ -0,0 +1,212 @@
+---
+sidebar_position: 1
+pagination_next: guides/getting-started/beginning-tutorial
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Prerequisites
+
+The first step is to install [Rust] and system dependencies. Keep in mind that this setup is only needed for _developing Tauri apps_. Your end-users are not required to do any of this.
+
+## Setting Up Windows
+
+:::info
+For those using the Windows Subsystem for Linux (WSL), please refer to our [Linux specific instructions](#setting-up-linux) instead.
+:::
+
+On Windows, go to [https://www.rust-lang.org/tools/install][install rust] to install `rustup` (the Rust installer). You also need to install Microsoft Visual Studio C++ build tools. The easiest way is to install [Build Tools for Visual Studio 2019]. When asked which workloads to install, ensure "C++ build tools" and the Windows 10 SDK are selected.
```

Comment:
Since the link will download version 2022 now.
```suggestion
On Windows, go to [https://www.rust-lang.org/tools/install][install rust] to install `rustup` (the Rust installer). You also need to install Microsoft Visual Studio C++ build tools. The easiest way is to install [Build Tools for Visual Studio 2022]. When asked which workloads to install, ensure "C++ build tools" and the Windows 10 SDK are selected.
```